device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -721,30 +721,32 @@ void startTwinInternal(IotHubEventCallback twinStatusCallback, Object twinStatus
      * @throws IllegalArgumentException if either callback is null
      */
     void startTwinInternal(IotHubEventCallback twinStatusCallback, Object twinStatusCallbackContext,
-                           TwinPropertiesCallBack genericPropertiesCallBack, Object genericPropertyCallBackContext)
+                           TwinPropertiesCallback genericPropertiesCallBack, Object genericPropertyCallBackContext)
             throws IOException, IllegalArgumentException, UnsupportedOperationException
     {
         if (!this.deviceIO.isOpen())
         {
-            //Codes_SRS_INTERNALCLIENT_34_081: [If device io has not been opened yet, this function shall throw an IOException.]
```

Comment:
You didn't need to include these. We've long since phased out of adding these kinds of comments, so I got rid of them for you.